fish_prompt.fish

```diff
@@ -76,18 +76,22 @@ function __bobthefish_escape_regex -a str -d 'A backwards-compatible `string esc
 end
 
 function __bobthefish_git_branch -S -d 'Get the current git branch (or commitish)'
-    set -l ref (command git symbolic-ref HEAD 2>/dev/null)
+    set -l branch (command git symbolic-ref HEAD | cut -d "/" -f3 2>/dev/null)
```

Comment:
Hmmm, just spotted a bug. This will ruin branches like `hotfix/something/really-critical`. I'll figure out a better way soon.

Reply:
```fish
command git symbolic-ref HEAD | string replace -r '^refs/heads/' ''
```

Reply:
![image](https://user-images.githubusercontent.com/14284827/96389316-6433f900-120b-11eb-9fdb-c771d99edf6c.png)
Fixed in 960241b.